Sources/CircularProgress/protocol/MouseTrackable.swift

```diff
@@ -0,0 +1,25 @@
+#if os(macOS)
```

Comment:
This is moot. This package is only for macOS.

Reply:
Yepp. Ill remove it. "Copy-pasta artefact"

Reply:
✅